deploy/helm/sumologic/values.yaml

```diff
@@ -442,6 +442,25 @@ sumologic:
       ## Matching unit will be excluded from Sumo. The logs will still be sent to logs metadata provider (otelcol).
       excludeUnitRegex: ""
 
+    otelcol:
+      ## additional exporters which may be used in any of the pipeline
+      ## remember add them to containers, systemd or kubelet routing as well
+      extraExporters: {}
```

Comment:
how does one specify which pipeline this extra exporter would be used in. For ex. containers or systemd?

Reply:
> additional exporters which may be used in any of the pipeline
this tells me, that they would consider logs from all the pipelines

> remember add them to containers, systemd or kubelet routing as well
not sure I understand this comment.

Reply:
> remember add them to containers, systemd or kubelet routing as well

This is leftover from first iteration

They cannot choose the pipeline. This is to simplify the feature and I think we should aim to unify the pipelines in the future